test/fixtures/gatherers/fstab.valid

```diff
@@ -0,0 +1,8 @@
+# test fstab
+
+/dev/system/root  /                       btrfs  defaults                      0  0
+/dev/system/root  /root                   btrfs  subvol=/@/root                1  1
+/dev/system/root  /home                   btrfs  subvol=/@/home                1  0
+/dev/system/swap  swap                    swap   defaults                      0  0
+/dev/system/root  /.snapshots             btrfs  subvol=/@/.snapshots          0  1
+UUID=DA2F-21CF    /boot/efi               vfat   utf8          
```

Comment:
Is this correct? The last entry doesn't have the `Backup` and `Order`?
I see that in the tests they are set to `0` both of them

Reply:
Auto-answer: Yes, they are optional, and they default to 0 if they are not present:
https://www.man7.org/linux/man-pages/man5/fstab.5.html